README.rst

```diff
@@ -0,0 +1,254 @@
+               |
+               | results/plot_pt.pdf
+               | results/plot_eta.pdf
+               |
+               V
+              STOP
+
+We shall see below how this sequence of commands is represented for the REANA
+serial workflow engine.
+
+
+5. Output results -
+-----------------------------
+
+
+The test run will create `ROOT <https://root.cern.ch/>`_ output files that
+usually contain histograms.
+
+.. code-block:: console
+
+   $ ls -l AnalysisResults.root
+   -rw-r--r-- 1 root root  31187 July 18 17:35 AnalysisResults.root
+
+The user typically uses the output files to produce final plots. For example,
+running ``plot.C`` output macro on the ``AnalysisResults.root`` output file will
+permit to visualise the pt distribution of the accepted events:
+
+.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-alice-pt-analysis/blob/master/docs/plot_pt.pdf
+   :alt: plot_pt.pdf
+   :align: center
+
+.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-alice-pt-analysis/blob/master/docs/plot_eta.pdf
+   :alt: plot_eta.pdf
+   :align: center
+
+
+
+Running the example on REANA cloud
+==================================
+
+We start by creating a `reana.yaml <reana.yaml>`_ file describing the above
+analysis structure with its inputs, code, runtime environment, computational
+workflow steps and expected outputs:
+
+.. code-block:: yaml
+
+    version: 0.3.0
+    inputs:
+      files:
+        - fix-env.sh
+        - data.txt
+        - AliAnalysisTaskEx01.cxx
+        - AliAnalysisTaskEx01.h
+        - runEx01.C
+        - plot.C
+        - OCDB.root
+      parameters:
+        data_location: http://opendata.cern.ch/record/1102/files/assets/alice/2010/LHC10h/000139038/ESD/0001/AliESDs.root
+    workflow:
+      type: serial
+      specification:
+        steps:
+          - environment: 'reanahub/reana-env-aliphysics:vAN-20180614-1'
+            commands:
+            - mkdir data/ && wget ${data_location} && mv AliESDs.root data/
+            - source fix-env.sh && root -b -q './runEx01.C' | tee run.log
+            - mkdir results && source fix-env.sh && root -b -q './plot.C'
+    outputs:
+      files:
+        - AnalysisResults.root
+        - results/plot_pt.pdf
+        - results/plot_eta.pdf
+
+
+We can now install the REANA command-line client, run the analysis and download the resulting plots:
+
+.. code-block:: console
+
+    $ # create new virtual environment
+    $ virtualenv ~/.virtualenvs/myreana
+    $ source ~/.virtualenvs/myreana/bin/activate
+    $ # install REANA client
+    $ pip install reana-client
+    $ # connect to some REANA cloud instance
+    $ export REANA_SERVER_URL=https://reana.cern.ch/
+    $ export REANA_ACCESS_TOKEN=XXXXXXX
+    $ # create new workflow
+    $ reana-client create -n my-analysis
+    $ export REANA_WORKON=my-analysis
+    $ # upload input code and data to the workspace
+    $ reana-client upload ./AliAnalysisTaskEx01.cxx \
```

Comment:
If all the inputs are specified in `reana.yaml`, you can use simply `upload` without any arguments, it'll upload all specified files.